addons/mail/static/src/js/tours/mail.js

```diff
@@ -22,9 +22,12 @@ tour.register('mail_tour', {
         actions.text("SomeChannel_" + t, this.$anchor.find("input"));
     },
 }, {
-    trigger: ".ui-autocomplete .ui-menu-item > a:contains(Private)",
-    content: _t("<p> Create a private channel.</p>"),
-    position: 'bottom',
+    trigger: ".ui-autocomplete:has(a:contains(Create))",
+    content: _t("<p> Create a public or private channel.</p>"),
```

Comment:
I'm not expert in translations but I believe something has to be added in pot/po files when adding new strings in stable.